examples/step-79/doc/intro.dox

```diff
@@ -566,26 +566,26 @@ further away from the set of points that satisfy the constraints -- or the other
 
 Here, we use an exact $l_1$ merit function to test the steps:
 @f{multiline}{
-  \phi(\textbf{x},\textbf{y}) = \int_{\partial \Omega} \textbf{u}\cdot
-  \textbf{t} - \alpha \int_\Omega (\log(s_1) + \log(s_2)) + p \sum_i\left|
-  \int_\Omega y_{2,i}(H(\sigma) - \rho) \right|
+  \phi(\mathbf{x},\mathbf{y}) = \int_{\partial \Omega} \mathbf{u}\cdot
+  \mathbf{t} - \alpha \int_\Omega (\log(s_1) + \log(s_2)) + p \sum_i\left|
+  \int_\Omega y_{2,i}(H(\varrho) - \rho) \right|
 \\
-+ p \sum_i\left| \int_{\partial \Omega} \textbf{y}_{1,i}\cdot \textbf{t}  -
-\int_\Omega \rho^p[\lambda \nabla \cdot \textbf{u} \nabla \cdot \textbf{y}_{1,i}
- + \mu \varepsilon{\textbf{u}}\varepsilon{\textbf{y}_{1,i}}]\right|
++ p \sum_i\left| \int_{\partial \Omega} \mathbf{y}_{1,i}\cdot \mathbf{t}  -
+\int_\Omega \rho^p[\lambda \nabla \cdot \mathbf{u} \nabla \cdot \mathbf{y}_{1,i}
+ + \mu \boldsymbol{\varepsilon}{\mathbf{u}}\boldsymbol{\varepsilon}{\mathbf{y}_{1,i}}]\right|
 \\
-+ p \sum_i\left| \int_\Omega z_{1,i}(s_1 - \sigma) \right|+ p \sum_i\left|
-\int_\Omega z_{2,i}(1-\sigma - s_2) \right|
++ p \sum_i\left| \int_\Omega z_{1,i}(s_1 - \varrho) \right|+ p \sum_i\left|
+\int_\Omega z_{2,i}(1-\varrho - s_2) \right|
 @f}
 
 Here, $p$ is a penalty parameter. This merit function being exact means that
 there exists some $p_0$ so that for any $p > p_0$, the merit function has its
 minima at the same location as the original problem. This penalty parameter is
 updated (by recommendation of Nocedal and Wright @cite Benson2002) as follows:
 @f[
-  p > \frac{0.5 x^T H x - x^t\nabla f}{\norm{c_i}_{l_\infty}, i \in \mathcal{E}}
+  p > \frac{\frac{1}{2} \mathbf{x}^T \cdot \mathbf{H} \cdot \mathbf{x} - \mathbf{x}^T \cdot \nabla f}{\norm{c_i}_{l_\infty}, i \in \mathcal{E}}
```

Comment:
Since `x` is the "vector of decision variables", it should be bold format. I believe the Hessian would also be a tensor / full matrix and so should also be bold, right?